assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm

```diff
@@ -193,18 +193,12 @@
 	</form>
 	<script type="text/JavaScript">
 		<!--
-			function toggleSelectAll(caller, elementName)
+			function toggleSelectAll(caller)
 			{
 				var newValue = caller.checked;
-				var elements = document.getElementsByName(elementName);
-				if(elements)
-				{
-					//SAK-19147 don't toggle last "Save all submissions in one folder"
-					for(var i = 0; i < elements.length -1; i++)
-					{
-						elements[i].checked = newValue;
-					}
-				}
+				$('.toggleSelectedAll').each(function(){
+                    $(this).prop('checked', newValue);
```

Comment:
Yes !! Sorry, I didn't notice !! :disappointed: 
I'll try to fix it in the branch and squash commits without removing this PR (playing with git :stuck_out_tongue_winking_eye:)